README.md

```diff
@@ -146,7 +146,7 @@ func main() {
 	}()
 
 	// send a channel update request to the other channel peer(s)
-	err = ch.Update(ctx, func(s *channel.State) error {
+	err = ch.Update(ctx, func(s *channel.State)) error {
```

Comment:
With the updated API, this should be `err = ch.Update(ctx, func(s *channel.State) {`